source/docs/advanced/features/debugging.md

```diff
@@ -0,0 +1,98 @@
+---
+id: debugging
+title: Debugging
+description: Debug your Front-Commerce app.
+---
+
+# Debugging
+
+Debugging your code is an important part of the development process. This documentation explains how you can debug your Front-Commerce frontend and backend code using either the [VS Code debugger](https://code.visualstudio.com/docs/editor/debugging) or [Chrome DevTools](https://developers.google.com/web/tools/chrome-devtools).
+
+Any debugger that can attach to Node.js can also be used to debug a Front-Commerce application. You can find more details in the Node.js [Debugging Guide](https://nodejs.org/en/docs/guides/debugging-getting-started/).
+
+## Debugging with VS Code
+
+Create a file named `.vscode/launch.json` at the root of your project with the following content:
+
+```json
+{
+  "version": "0.2.0",
+  "configurations": [
+    {
+      "name": "FC: debug server-side",
+      "type": "node-terminal",
+      "request": "launch",
+      "command": "npm run start -- --inspect"
+    },
+    {
+      "name": "FC: debug client-side",
+      "type": "pwa-chrome",
+      "request": "launch",
+      "url": "http://localhost:4000"
+    },
+    {
+      "name": "FC: debug full stack",
+      "type": "node-terminal",
+      "request": "launch",
+      "command": "npm run start -- --inspect",
+      "console": "integratedTerminal",
+      "serverReadyAction": {
+        "pattern": "started server on .+, url: (https?://.+)",
+        "uriFormat": "%s",
+        "action": "debugWithChrome"
+      }
+    }
+  ]
+}
+```
+
+If you're [changing the port number](/docs/reference/environment-variables.html#Host) your application starts on, replace the `3000` in `http://localhost:3000` with the port you're using instead, you can use the the url defined in your `FRONT_COMMERCE_URL`
+
+Now in vscode go to the Debug panel (<kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>D</kbd> on Windows/Linux, <kbd>⇧</kbd>+<kbd>⌘</kbd>+<kbd>D</kbd> on macOS), select a launch configuration, then press <kbd>F5</kbd> or select **Debug: Start Debugging** from the Command Palette to start your debugging session.
+
+## Debugging with Chrome DevTools
+
+### Client-side code
+
+Start your development server as usual by running `front-commerce start`, `npm run start`. Once the server starts, open `http://localhost:4000` (or your alternate URL) in Chrome. Next, open Chrome's Developer Tools (<kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>J</kbd> on Windows/Linux, <kbd>⌥</kbd>+<kbd>⌘</kbd>+<kbd>I</kbd> on macOS), then go to the **Sources** tab.
```

Comment:
> also it would be nice to not be 100% Chrome oriented and to mention that it also works well for client-side with Firefox 
I think this is related to what we have discussed a retro, we support a set of conditions and environments and anything added to that is additional, this is only as a point of reference to debug, based on the majority use case, although not limited to chrome, using chrome as reference supports a wider base of browsers/users (chromium).

And because for the server-side code we also supply docs for chrome dev tools node debugger, it makes sense to include it for the front end as well.

Reply:
I did add some reference to other clients and also inspector clients with links to docs.